python/ql/test/library-tests/regexparser/test.py

```diff
@@ -0,0 +1,2 @@
+import re
+baz = re.compile(r'\+0')
```

Comment:
How about moving this into the `regexparser/unittests.py` file?

Reply:
Yeah, the current motley crew of test files could be cleaned up...